CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ Package-specific changes not released in any SDK will be added here just before
 
 ## Unpublished
 
+- Added support for the light sensor. ([#18177](https://github.com/expo/expo/pull/18177) by [bearkillerPT](https://github.com/bearkillerPT))
+
```

Comment:
Please delete entry from this CHANGELOG. It will be added automatically here during release. Leave only the one in `packages/expo-sensors/CHANGELOG.md`
```suggestion
```